00-HTML-CSS-basics/0.1.v.html

```diff
@@ -0,0 +1,18 @@
+<html>
+  <head>
+    <title>Multimedia</title>
+  </head>
+  <body>
+    <img src="http://diveintohtml5.info/i/openclipart.org_johnny_automatic_monkey_reading.png" />
```

Comment:
Just a suggestion: when you have an ``img`` tag, you should always add an *alt* attribute with a string to describe the image in case that the content can't be loaded by the browser, you can do this like this:
````
<img src:"http://diveintohtml5.info/i/openclipart.org_johnny_automatic_monkey_reading.png" alt="Johnny automatic monkey reading image" />
````